modules/persistent-storage-csi-azure-file-cross-sub-dynamic-provisioning-procedure.adoc

```diff
@@ -0,0 +1,106 @@
+// Module included in the following assemblies:
+//
+// * storage/container_storage_interface/persistent_storage-csi-azure-file.adoc
+//
+:_mod-docs-content-type: PROCEDURE
+[id="persistent-storage-csi-azure-file-cross-sub-dynamic-provisioning-procedure_{context}"]
+= Dynamic provisioning across subscriptions for Azure File
+
+.Prerequisites
+* Installed Azure cluster with the service principal or managed identity as Azure identity
+
+* Access to another subscription in the tenant with the cluster  
+
+* Logged in to the Azure CLI
+
+.Procedure
+To use Azure File dynamic provisioning across subscriptions:
+
+. Record the Azure identity (service principal or managed identity) by running the following applicable commands. The Azure identity is needed in the next step:
++
+* If using the _service principal_ as Azure identity when installing the cluster:
++
+[source,terminal]
+====
+$ sp_id=$(oc -n openshift-cluster-csi-drivers get secret azure-file-credentials -o jsonpath='{.data.azure_client_id}' | base64 --decode)
+
+$ az ad sp show --id ${sp_id} --query displayName --output tsv
+====
++
+* If using the _managed identity_ as Azure identity when installing the cluster:
++
+[source,terminal]
+====
+$ mi_id=$(oc -n openshift-cluster-csi-drivers get secret azure-file-credentials -o jsonpath='{.data.azure_client_id}' | base64 --decode)
+
+$ az identity list --query "[?clientId=='${mi_id}'].{Name:name}" --output tsv
+====
+
+. Obtain the Azure identity (service principal or managed identity) permission to access the resource group in another subscription where you want to provision the Azure File share:
```

Comment:
Do we need (service principal or managed identity) here, as it is in the prereqs?

```suggestion
. Record the Azure identity by running the following applicable commands. The Azure identity is needed in the next step:
```